.travis.yml

```diff
@@ -22,10 +22,21 @@ jobs:
       env: TOX_ENV=py35-full
     - python: '3.6'
       env: TOX_ENV=py36-full
+    - python: '3.6'
+      arch: arm64
+      env: TOX_ENV=py36-full
     - python: '3.7'
       env: TOX_ENV=py37-full
+    - python: '3.7'
+      arch: arm64
+      env: TOX_ENV=py37-full
+    - python: '3.8'
+      service: docker
```

Comment:
It looks like this should be `services`, not `service` (although it seems to work anyway?)

Reply:
Yes, my bad. Corrected the same.